examples/pipeline_tacotron2/README.md

```diff
@@ -42,4 +42,39 @@ be in `./logs`.
 If `./ckpt.pth` already exist, this script will automatically load the file and try to continue
 training from the checkpoint.
 
-This command takes around 36 hours to train on 8 NVIDIA Tesla V100 GPUs.
\ No newline at end of file
+This command takes around 36 hours to train on 8 NVIDIA Tesla V100 GPUs.
+
+## Training Tacotron2 with phoneme as input
+
+#### Dependencies
+
+If you want to use [DeepPhonemizer](https://github.com/as-ideas/DeepPhonemizer) as
```

Comment:
Since this section is about the use of phoneme and there is no alternative prepared at the moment, the wording can be improved.

Reply:
I've rewrite the description [here](https://github.com/pytorch/audio/pull/1668/files#diff-a17fd0fdb11c8336af63efc614215c0ef98e67d810b77c5c5e8d87a012ca64f1R76).
Thanks.